src/docker-phpunit-command.js

```diff
@@ -18,6 +39,13 @@ module.exports = class DockerPhpUnitCommand extends RemotePhpUnitCommand {
     }
 
     wrapCommand(command) {
-        return `${this.dockerCommand} ${command}`;
+        var runCommand = `${this.dockerCommand}`;
+        runCommand += `${this.configuration}`;
+        runCommand += ' ' + `${this.file}`;
+        if (this.filter) {
+            runCommand += ' ' + `${this.filter}`;
+        }
+
+        return runCommand;
```

Comment:
This block also makes me nervous because we are no longer actually "wrapping" the command - you are re-constructing it, so changes that are made to the base command generator will not trickle down and impact docker users.